indigo/distribution.yaml

```diff
@@ -12684,6 +12684,21 @@ repositories:
       url: https://github.com/turtlebot/turtlebot_create_desktop.git
       version: indigo
     status: maintained
+  turtlebot_exploration_3d:
+    doc:
+      type: git
+      url: https://github.com/RobustFieldAutonomyLab/turtlebot_exploration_3d.git
+      version: indigo
+    release:
+      tags:
+        release: release/indigo/{package}/{version}
+      url: https://github.com/RobustFieldAutonomyLab/turtlebot_exploration_3d.git
+      version: 0.0.1
```

Comment:
As I said, if you just want to add documentation, do not put in a release section. If you want to release it, so binary packages are built, you need to run bloom and follow the appropriate section in the "Get Involved" wiki page:

http://wiki.ros.org/Get%20Involved#Release_Binary_Packages